compile_to_js.sh

```diff
@@ -0,0 +1,475 @@
+#!/bin/bash
+
```

Comment:
This one should go into `Distributions/JavaScript` directory

Reply:
done